scripts/sign

```diff
@@ -7,6 +7,15 @@ sign_and_submit() {
     (source AMOKEYS && (web-ext sign -s build --api-key $AMOKEY --api-secret $AMOSECRET || true))
 }
 
+publish_beta_nonewtab() {
+    npm run clean
+    npm run build
+    scripts/version.js beta
+    sed 's/tridactyl.vim@cmcaine/tridactyl.vim.betas.nonewtab@cmcaine/' -i build/manifest.json
+    sed '/"newtab": "static\/newtab.html"/d' -i build/manifest.json
```

Comment:
I'd make this a little more robust - [any whitespace]*newtab:*/d for example

Reply:
Fixed it